.gitignore

```diff
@@ -0,0 +1,4 @@
+Web Experiences/Blueprints/email-capture-corner/.DS_Store
+Web Experiences/Blueprints/.DS_Store
+.DS_Store
+Web Experiences/Blueprints/.DS_Store
```

Comment:
We need to remove all .DS_Store files in the project before implementing the .gitignore
Also just ".DS_Store" will have the same effect, all other lines are unnecessary.

Reply:
I removed the git ignore file, once everything is added to master we immediately check if there is any DS remaining, if none will implement the gitignore file again.